staging/kos/pkg/apis/network/types.go

```diff
@@ -177,9 +177,57 @@ type SubnetSpec struct {
 	VNI uint32
 }
 
+type SubnetConditionType string
+
+type ConditionStatus string
+
+const (
+	// ConditionTrue is the condition status which means a resource is in the
+	// condition.
+	ConditionTrue ConditionStatus = "True"
+
+	// ConditionFalse is the condition status which means a resource is not in
+	// the condition.
+	ConditionFalse ConditionStatus = "False"
```

Comment:
I suggest we import and use core/v1's ConditionStatus rather than define our own variant.